src/crypto-sha1-2-browser.js

```diff
@@ -1,18 +1,19 @@
+/* global self */
+
 'use strict'
 
 const nodeify = require('nodeify')
 
 const webCrypto = getWebCrypto()
 
 function getWebCrypto () {
```

Comment:
This file (`crypto-sha1-2-browser.js`) adds a new method `sha3` that is not present in `crypto-browser.js`, other than that they are exactly the same, should we remove `crypto-browser.js`?